### PR TITLE
Convert python scripts to python3

### PR DESCRIPTION
**Pull request by shenitzky**
A series of patches to support python-3 for running the DR in RHEL-8 host.
@sandrobonazzola @machacekondra @mwperina

### CODE REVIEW

File: files/generate_vars.py [hunk 1]
```diff
@@ -181,10 +180,8 @@ def _validate_output_file_exists(self, fname, log):
                         sys.exit(0)
```
Comment (barpavel) [inline files/generate_vars.py]: Is it me or the 2 lines are not properly indented?
sys.exit(0)
break

**Comment by mwperina** [comment]
Unfortunately this is beyond my Python skills, please make sure that those scripts are compatible with both Python 2 (EL7) and Python 3 (EL8/Fedora), because we need to support for both versions

File: files/generate_vars.py (hunk 1 above)
Comment (aabawer) [reply files/generate_vars.py]: seems okay. break belongs to outer if. if you exit there is no need for break after it.

File: files/ovirt-dr [hunk 2]
```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/python
-from ConfigParser import SafeConfigParser
+try:
```
Comment (barpavel) [inline files/ovirt-dr]: Wouldn't it be better to ask if we are Python 2 or 3 (if six.PY3) and import the relevant version?
The same for other 4 modules.

Comment (shenitzky) [reply files/ovirt-dr]: Both methods are fine.
Will stick with this one.

**Comment by shenitzky** [comment]
Travis failed on style issues that fixed by another pull request

**Pull request merged**